pallets/liquidation-pools/src/lib.rs

```diff
@@ -326,6 +333,25 @@ pub mod module {
 			Self::deposit_event(Event::LiquidationPoolsBalanced);
 			Ok(().into())
 		}
+
+		/// Updates balance of liquidation pool
+		/// - `pool_id`: PoolID to update
+		/// - `by_amount`: value to change the balance. If positive do add, else do remove.
+		#[pallet::weight(10_000)] // Stub, function is for dev purposes only.
+		#[transactional]
+		pub fn update_balance(
+			origin: OriginFor<T>,
+			pool_id: CurrencyId,
+			by_amount: Amount,
```

Comment:
We usually omit grammar suffix like this. 
```suggestion
			amount: Amount,
```